orion/primitives/azure_anomaly_detector.py

```diff
@@ -130,6 +132,9 @@ def detect_anomalies(X, index, interval, overlap_size, subscription_key, endpoin
             Advanced model parameter, between 0-99, the lower the value is, the larger
             the margin value will be which means less anomalies will be accepted. If not given,
             `None` is used.
+        timezone (str):
```

Comment:
Can you describe the format of the `str` to define the timezone? It looks you are using pytz, so maybe a reference link here would be helpful.

Reply:
Good suggestion, I added a link to `pytz`. I did not insert a specific format since there are many available by `pytz`.